extensions/tika/runtime/src/main/java/io/quarkus/tika/runtime/TikaConfiguration.java

```diff
@@ -12,11 +13,37 @@
 @ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
 public class TikaConfiguration {
     /**
-     * The path to the tika-config.xml
+     * Path to the {@code tika-config.xml}
      */
     @ConfigItem
     public Optional<String> tikaConfigPath;
 
+    /**
+     * Comma separated list of the parsers which must be supported.
+     * <p>
+     * Most of the document formats recognized by Apache Tika are supported by default but it affects
+     * the application memory and native executable sizes. One can list only the required parsers in
+     * {@code tika-config.xml} to minimize a number of parsers loaded into the memory, but using this
+     * property is recommended to achieve both optimizations.
+     * <p>
+     * Either the abbreviated or full parser class names can be used.
+     * At the moment only PDF parser can be listed using a reserved 'pdf' abbreviation.
+     * Custom class name abbreviations have to be used for all other parsers.
+     * For example:
+     * 
```

Comment:
The formatter does not let me add the paragraph tags around the `pre` ones, it removes them, I guess they are not needed with the `pre` tags